airflow/providers/google/cloud/operators/sftp_to_gcs.py

```diff
@@ -19,15 +19,23 @@
 This module contains SFTP to Google Cloud Storage operator.
 """
 import os
+import warnings
 from tempfile import NamedTemporaryFile
 from typing import Optional, Union
 
 from airflow.exceptions import AirflowException
 from airflow.models import BaseOperator
-from airflow.providers.google.cloud.hooks.gcs import GCSHook
-from airflow.providers.sftp.hooks.sftp import SFTPHook
 from airflow.utils.decorators import apply_defaults
 
+try:
+    from airflow.providers.google.cloud.hooks.gcs import GCSHook
+except ImportError:
+    warnings.warn("The package providers-google-cloud must be installed to use the operator.",
```

Comment:
Should it be `providers-ssh` instead of gcp?

Reply:
Ah yeah, that was one of the latest additions - I know detect such mistakes automatically (i.e no self-provider try/except is allowed) :). One of the 63 errors to fix :)